web-console/knikubevirt/Create-vm/step-4-advanced/wizard-advanced.md

```diff
@@ -0,0 +1,36 @@
+# Create A Virtual Machine
+
+## Step 4: Advanced
+
+The “Advanced” setting step of the create VM wizard includes:
+
+- Cloud-Init / Sysprep - determines the contents of these settings based on the OS that is selected. (The difference between the cloud-init and the sys-prep is basically the difference between windows and other OSs)
+- Virtual hardware
+
+### Cloud-Init / Sysprep
+
+The user has 2 options (radio buttons to choose from) to configure the "cloud-init/Sysprep" settings:
+
+1. by filling a form (default)
+2. by writing a custom script
+
+- SSH Authorized Keys: Enter a comma-separated list of public SSH keys that can be used for authentication.
+- NIC: Select the NIC that should communicate with the DNS servers.
+- DNS Servers: Enter a comma-separated list of DNS server IP addresses.
+- DNS Search Domains: Enter a comma-separated list of DNS search domain IP addresses.
+
+![empty form](img/form-empty.png)
+
+By the default the user fills out the fields in the form
+![user fills form](img/ssh-keys.png)
+
+In case the user chooses to use a custom script
+![user writes custom script](img/customscript.png)
```

Comment:
I think we'd still want to provide the Custom script label on the text area here to support accessibility.

Reply:
ah I actually meant the label not the placeholder text. So it should say Custom script above the textarea.